components/list/list-item-drag-image.js

```diff
@@ -84,7 +84,7 @@ class ListItemDragImage extends LocalizeCoreElement(SkeletonMixin(RtlMixin(LitEl
 				color: white;
 				left: 26px;
 				min-width: 1.4rem;
-				padding: 0.2rem 0.4rem;
+				padding: 0.25rem 0.4rem;
```

Comment:
This was the only case where the `line-height` was contributing to its height in a meaningful way. We want these to be circles when the counts are single digits. I've increased the padding to compensate.

<img width="902" alt="Screenshot 2025-04-23 at 1 24 27 PM" src="https://github.com/user-attachments/assets/b444e02f-4a20-484d-b95f-619c9648691f" />